orchagent/pfcqueue.h

```diff
@@ -0,0 +1,84 @@
+#ifndef PFC_QUEUE_H
+#define PFC_QUEUE_H
+
+#include <vector>
+
+extern "C" {
+#include "sai.h"
+}
+
+// PFC queue interface class
+// It resembles RAII behavior - pause storm is mitigated (queue is locked) on creation,
+// and is restored (queue released) on removal
+class PfcQueue
```

Comment:
As we discussed, shall we change the PfcQueue to PfcActionHandler or some other name you preferred?

Reply:
done